### PR TITLE
Update typescript definition for selectize to 0.12.13

### PR DESCRIPTION
**Pull request by naBausch**
case 2. Improvement to existing type definition.
- https://github.com/selectize/selectize.js/blob/master/docs/api.md#methods_other

### CODE REVIEW

File: selectize/selectize.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for Selectize 0.11.2
+// Type definitions for Selectize 0.12.13
 // Project: https://github.com/brianreavis/selectize.js
 // Definitions by: Adi Dahiya <https://github.com/adidahiya>
+// Definitions by: Natali Bausch <https://github.com/naBausch>
```
Comment (adidahiya) [inline selectize/selectize.d.ts]: please include your name on the above line, in a list, like this:

```
 // Definitions by: Adi Dahiya <https://github.com/adidahiya>, Natali Bausch <https://github.com/naBausch>
```

File: selectize/selectize.d.ts [hunk 2]
```diff
@@ -562,6 +563,12 @@ declare namespace Selectize {
          * Clears the render cache. Takes an optional template argument (e.g. "option", "item") to clear only that cache.
          */
         clearCache(template?: string): void;
+        
+        /**
+         * When the `settings.placeholder` value is changed, the new placeholder will be displayed.
+         */
+        updatePlaceholder: () => void;
+
```
Comment (adidahiya) [inline selectize/selectize.d.ts]: minor: extra newline

File: selectize/selectize.d.ts [hunk 3]
```diff
@@ -562,6 +563,12 @@ declare namespace Selectize {
          * Clears the render cache. Takes an optional template argument (e.g. "option", "item") to clear only that cache.
          */
         clearCache(template?: string): void;
+        
+        /**
+         * When the `settings.placeholder` value is changed, the new placeholder will be displayed.
+         */
+        updatePlaceholder: () => void;
```
Comment (adidahiya) [inline selectize/selectize.d.ts]: for consistency, I think this should be written as:

```ts
updatePlaceholder(): void;
```

also, are you sure the return type is void? I know the docs don't say, but how does it work at runtime?

**Review by adidahiya — CHANGES_REQUESTED** [review]

File: selectize/selectize.d.ts (hunk 1 above)
Comment (naBausch) [inline selectize/selectize.d.ts]: Done 👍

File: selectize/selectize.d.ts (hunk 2 above)
Comment (naBausch) [inline selectize/selectize.d.ts]: Done 👍

**Comment by naBausch** [comment]
@adidahiya thanks for your feedback please see the updated version

**Review by adidahiya — APPROVED** [review]

**Comment by adidahiya** [comment]
thanks, lgtm @naBausch 👍

**Comment by jantimon** [comment]
How do we trigger a new release for https://www.npmjs.com/package/@types/selectize ?

**Pull request merged**